src/utilities/getSmallestNumber/getSmallestNumber.test.js

```diff
@@ -0,0 +1,126 @@
+import { getSmallestNumber } from './getSmallestNumber';
+
+describe('getSmallestNumber()', () => {
+  describe('when provided with valid arrays of numbers', () => {
+    describe('and array contains positive and negative numbers', () => {
+      it('should return -5 for [2, -5, 10, 1, 4]', () => {
```

Comment:
Personally, I wouldn't include the exact value in the test description, especially if it is a complex value. I would just do something like that:
```
describe('and array contains positive and negative numbers', () => {
 it('should return the correct number', () => {
    ... 
 });
});
```

This goes for all your tests in this PR.